src/pages/Main.ts

```diff
@@ -19,6 +18,7 @@ import { TitleManager } from 'src/components/TitleManager';
 import ImportStatus from 'src/data/ImportStatus';
 import { getOidcConfig } from 'src/libs/auth';
 import { getEnabledBrand } from 'src/libs/brand-utils';
+import { reportError } from 'src/libs/error';
 import { PageViewReporter } from 'src/libs/events';
 import { LocationProvider, PathHashInserter, Router } from 'src/libs/nav';
 
```

Comment:
can we get Main() signature to have the React FC type signature we agreed on?  Should be a very quick fix.

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4247/commits/2502dfdaf14826300da1e65820909b7cf06b81b8.